tools/wheels/cibw_test_command.sh

```diff
@@ -4,10 +4,15 @@
 set -xe
 
 PROJECT_DIR="$1"
-UNAME="$(uname)"
 
 python -c "import numpy; numpy.show_config()"
-python -c "import sys; import numpy; sys.exit(not numpy.test('full', extra_argv=['-vv']))"
+if [[ $RUNNER_OS == "Windows" ]]; then
+    # FIXME: Don't skip tests
+    # python -c "import sys; import numpy; sys.exit(not numpy.test('full', extra_argv=['-k not test_build_import', '-v']))"
+    PY_DIR=$(python -c "import sys; print(sys.prefix)")
+    mkdir $PY_DIR/libs
```

Comment:
The failing test was trying to write to a missing directory. ``sys.prefix`` exists but the libs folder under it doesn't. This feels very hacky, though.

Reply:
Agreed. That test is a total hack.

Reply:
Could you mention issue 20391 here so we can remove this once that is resolved?